app/src/main/java/fragments/TempFeedFragment.java

```diff
@@ -62,6 +81,16 @@ public class TempFeedFragment extends Fragment implements AdapterView.OnItemSele
     public PostAdapter adapter;
     SwipeRefreshLayout swipeContainer;
 
+    // Weather data
+    private String description;
+    private String temperature;
+    // sunset time of local timezone
+    private String sunsetTime;
+    long myTimeZone = -25200;
+
+    private final String url = "https://api.openweathermap.org/data/2.5/weather";
```

Comment:
since you have a lot of these magic strings/ints/values a pattern that i've used before is created a AppConstants.java class that purely contains all of these values. That way you can also reuse them in other classes if you want to.

eg
AppConstants.java {
public static final String weatherUrl = "https://api.openweathermap.org/data/2.5/weather";
}

then you can use AppConstants.weatherUrl anywhere youd like.